sites/partners/package.json

```diff
@@ -8,13 +8,15 @@
   "scripts": {
     "dev": "NODE_OPTIONS='--inspect=9231' next -p ${NEXTJS_PORT:-3001}",
     "build": "next build",
-    "test": "concurrently \"yarn dev\" \"cypress open\"",
-    "test:headless": "concurrently \"yarn dev\" \"cypress run\"",
+    "cypress:open": "wait-on \"http-get://localhost:${PORT:-3100}/docs\" \"http-get://localhost:${PORT:-3001}\" && cypress open",
```

Comment:
Are you going to add some commands to the top level package.json?

Reply:
Thanks for commenting - there was a change I had to make to simplify the top-level package.json. But there aren't any new commands to add to the top-level package.json; the new cypress:open and cypress:run are just for use here.

The overall approach here: let sites/partners/package.json worry about all of the things that need to be running for this test to work. That way in the top-level package.json, we can just cd into sites/partners and run yarn:test.